lib/network_monitor.c

```diff
@@ -95,6 +95,51 @@ struct mptcpd_nm
         bool monitor_loopback;
 };
 
+// -------------------------------------------------------------------
+//                         Helper Functions
+// -------------------------------------------------------------------
+
+/**
+ * @brief Wrap different versions of ELL @c l_netlink_send().
+ *
+ * ELL 0.68 changed the API for @c l_netlink_send().  This helper
+ * function wraps the two different function calls so that mptcpd will
+ * work with both pre- and post-0.68 @c l_netlink_send() APIs.
+ */
+static unsigned int netlink_send(struct l_netlink *netlink,
+                                 uint16_t type,
+                                 uint16_t flags,
+                                 void const *data,
+                                 uint32_t len,
+                                 l_netlink_command_func_t function,
+                                 void *user_data,
+                                 l_netlink_destroy_func_t destroy)
+{
+#ifdef HAVE_L_NETLINK_MESSAGE_NEW_SIZED
+        // ELL >= 0.68
+        struct l_netlink_message *const message =
+                l_netlink_message_new_sized(type, flags, len);
```

Comment:
It looks like ELL implicitly deallocates this `l_netlink_message` object after the `l_netlink_send()` call/command is done.  AFAICT, we do not have to explicitly call `l_message_unref()`.

Reply:
Indeed, the `destroy_command()` helper is now calling `l_netlink_message_unref(command->message);`